.github/workflows/windows-alt.yml

```
@@ -187,3 +187,37 @@ jobs:
       - name: x86_64-w64-mingw32 Build/Test
         run:
           ./tests/ci/run_cross_mingw_tests.sh x86_64 w64-mingw32 "-DCMAKE_BUILD_TYPE=Release"
+  mingw-clang:
+    if: github.repository_owner == 'aws'
+    runs-on: windows-latest
+    steps:
+      - name: Install MSYS2
+        uses: msys2/setup-msys2@v2
+        id: setup_msys2
+        with:
+          msystem: CLANG64 # x86_64 clang-based environment
```

```suggestion
          msystem: CLANG64 # x86_64 clang-based environment
          update: true
```
I'm so sorry, I forgot this option. otherwise, older versions of packages can be installed